libnavigation-core/src/main/java/com/mapbox/navigation/core/NavigationComponentProvider.kt

```diff
@@ -67,6 +68,16 @@ internal object NavigationComponentProvider {
     fun createBillingController(
         accessToken: String?,
         navigationSession: NavigationSession,
+        tripSession: TripSession,
+        arrivalProgressObserver: ArrivalProgressObserver
+    ): BillingController = BillingController(
+        navigationSession,
+        arrivalProgressObserver,
+        accessToken.toString(),
```

Comment:
just curious: `fun String?.toString` return "null" if a value is `null`. Is it expected? May be set explicitly null value instead of it?

Reply:
Yes, this is fine, the token is invalid either way.

Reply:
The native implementation needs a non-nullable object, that's why this was integrated.